vehicle/src/main/java/org/egov/vehicle/web/model/Vehicle.java

```diff
@@ -0,0 +1,119 @@
+package org.egov.vehicle.web.model;
+
+import org.egov.vehicle.web.model.user.User;
+import org.springframework.validation.annotation.Validated;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonValue;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.Setter;
+import lombok.ToString;
+
+@Validated
+@javax.annotation.Generated(value = "io.swagger.codegen.v3.generators.java.SpringCodegen", date = "2021-01-06T05:37:21.257Z[GMT]")
+@AllArgsConstructor
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Setter
+@ToString
+@Builder
+public class Vehicle {
+
+    @JsonProperty("id")
+    private String id = null;
+
+    @JsonProperty("tenantId")
+    private String tenantId = null;
+
+    @NonNull
+    @JsonProperty("registrationNumber")
+    private String registrationNumber  = null;
+
+    @NonNull
```

Comment:
Add max size annotation on all string fields in POJO

Reply:
Done